cs/mailing.texy

```diff
@@ -179,7 +192,28 @@ $mailer->send($mail);
 \--
 
 
-Můžeme si také vytvořit vlastní mailer, jde o objekt implementující rozhraní [Nette\Mail\IMailer |api:].
+Fallback Mailer
+---------------
+
+Máme-li občas potíže s odesíláním emailů z jednoho maileru (např. výpadek smtp serveru), můžeme využít `FallbackMailer`, ve kterém nastavíme seznam mailerů pro případ, že předchozí selže.
+
+`Fallback Mailer` nám dává několik funkcí navíc.
 
+- Využívat více mailerů.
+- Nastavit počet pokusů pro odeslání.
+- Nastavit čekací dobu mezi pokusy.
+
+/--php
+$mailers[] = $smtpMailer;
+$mailers[] = $backupSmtpMailer;
+$mailers[] = $sendmailMailer;
+
+$retryCount = 3;
+$retryWaitTime = 1000; // v milisekundách
+
+$mailer = new Nette\Mail\FallbackMailer($mailers, $retryCount, $retryWaitTime);
```

Comment:
Predal bych jenom pole maileru primo do konstruktoru a zbytek bych asi neuvadel. Jsou to stejne defaultni hodnoty. A nedelal bych mezi send mezirku ;-)